src/chunk.c

```diff
@@ -65,7 +66,7 @@ static void chunk_scan_ctx_init(ChunkScanCtx *ctx, Hyperspace *hs, Point *p);
 static void chunk_scan_ctx_destroy(ChunkScanCtx *ctx);
 static void chunk_collision_scan(ChunkScanCtx *scanctx, Hypercube *cube);
 static int	chunk_scan_ctx_foreach_chunk(ChunkScanCtx *ctx, on_chunk_func on_chunk, uint16 limit);
-static void chunk_show_chunks_first_call(FunctionCallInfo fcinfo);
+static Chunk **show_drop_chunks(Oid table_relid, Datum older_than_datum, Datum newer_than_datum, Oid older_than_type, Oid newer_than_type, char *caller_name, MemoryContext mctx, uint64 *num_chunks_returned);
```

Comment:
I don't think this function has anything to do with showing or dropping chunks. Let's have a name like `chunk_get_in_time_range` or something

Reply:
Renamed to `chunk_get_chunks_in_time_range`

Reply:
Renamed to `chunk_get_chunks_in_time_range`